shub/utils.py

```diff
@@ -163,16 +165,17 @@ def write_and_echo_logs(keep_log, last_logs, rsp, verbose):
             raise RemoteErrorException("Deploy failed: {}".format(last_log))
 
 
-def echo_short_log_if_deployed(deployed, last_logs, log_file, verbose):
+def echo_short_log_if_deployed(deployed, last_logs, keep_log, verbose):
```

Comment:
This is a backward-incompatible change. Can we restore backward compatibility? (i.e. add a new parameter, maybe log a deprecation warning if `log_file` is not `None` or similar)

Reply:
Restored backward compat, ended up moving `keep_log` out of this function anyway since it was simpler. also updated the tests to actually assert what was logged.

is a deprecation warning still necessary? I assumed this function was an internal utility and not user-facing